src/features/customerView/leasesCard/BerthLeasesCard.tsx

```diff
@@ -22,6 +23,7 @@ const BerthLeasesCard = ({ leases, handleShowContract }: BerthLeasesCardProps) =
       mooringType: lease.mooringType,
       link: lease.harbor ? `/harbors/${lease.harbor.id}` : undefined,
       address: [lease.harbor?.name || '', lease.pierIdentifier || '', lease.berthNum].filter(Boolean).join(' '),
+      renderContractDetails: (leaseId: string) => <BerthContractDetailsContainer leaseId={leaseId} />,
```

Comment:
If `handleShowContract` is no longer needed, you might consider removing it from `BerthLeasesCardProps`.

Reply:
Done